streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/impl/ApplicationServiceImpl.java

```diff
@@ -1115,17 +1120,28 @@ public Application getApp(Application appParam) {
       }
     }
 
-    if (!application.getHotParamsMap().isEmpty()) {
-      if (ExecutionMode.YARN_APPLICATION.equals(application.getExecutionModeEnum())) {
-        if (application.getHotParamsMap().containsKey(ConfigConst.KEY_YARN_APP_QUEUE())) {
-          application.setYarnQueue(
-              application.getHotParamsMap().get(ConfigConst.KEY_YARN_APP_QUEUE()).toString());
-        }
-      }
-    }
+    fillBackYarnQueue(application);
+
     return application;
   }
 
+  private void fillBackYarnQueue(Application application) {
+    if (!(ExecutionMode.YARN_APPLICATION == application.getExecutionModeEnum()
```

Comment:
AnchorA:   
The logic is meaningful for y-app mode & y perjob mode not for y-session mode.
The corresponding config items were set when users create flink yarn-session cluster.

Reply:
> AnchorA: The logic is meaningful for y-app mode & y perjob mode not for y-session mode. The corresponding config items were set when users create flink yarn-session cluster.

Yes,  as you said